services/api-server/src/simcore_service_api_server/models/schemas/studies.py

```diff
@@ -27,3 +31,9 @@ class StudyPort(solvers.SolverPort):
         "Correponds to the UUID of the parameter/probe node in the study",
         title="Key name",
     )
+
+
+class LogLinkMap(pydantic.BaseModel):
+    map: typing.Mapping[NodeName, DownloadLink] = pydantic.Field(
```

Comment:
Q: why you like to keep `pydantic.` prefix?
Everywhere else in this repo we import the classes directly as  `from pydantic import BaseModel, Field`. I do not see any risk of collision here and therefore is unnecessarily  verbose, right? :-)

Reply:
Q: what advantage do you see of `Mapping` vs simply `dict`. Effectively you will always be using here a `dict`

Reply:
Concerning the dict, I don't have a good reason. I agree it is probably better to simply to use `dict` here. 
I have corrected